docs/arcaflow/workflows/flow-control.md

```diff
@@ -1,12 +1,15 @@
 # Using Flow Control Mechanics
 
-Flow control allows the workflow author to build a workflow with a decision tree based on supported flow logic. These flow control operations are not implemented by plugins, but are part of the workflow engine itself.
+Flow control allows the workflow author to build a workflow with a decision tree based
+on supported flow logic. These flow control operations are not implemented by plugins,
+but are part of the workflow engine itself.
 
 ## Implicit Step Relationships
 
 Any time the input of a step relies on the output of another step via an [Arcaflow
-expression](http://127.0.0.1:8000/arcaflow/workflows/expressions/), an implicit step
-relationship is established. In this case, the Arcaflow engine holds the execution of the dependent step until the output from the supplier step is available.
+expression](expressions.md), an implicit step relationship is established. In this case,
+the Arcaflow engine holds the execution of the dependent step until the output from the
+supplier step is available.
```

Comment:
"dependent" to "supplier" sounds like an odd combination. Maybe "supplier" -> "consumer" to focus more on the data dependency?

Reply:
It may make sense to also explicitly say the step names so that the reader doesn't need to look through the workflow below to figure out which one is the dependent/comsumer step or supplier step.